bin/deploy-bamboo.sh

```
@@ -113,10 +133,16 @@ dockerRun() {
     -e "AWS_ACCESS_KEY_ID=$bamboo_AWS_ACCESS_KEY_ID" \
     -e "AWS_ACCOUNT=$bamboo_AWS_ACCOUNT" \
     -e "AWS_SECRET_ACCESS_KEY=$bamboo_AWS_SECRET_ACCESS_KEY" \
+    -e "BEDROCK_MODEL_ID=$bamboo_BEDROCK_MODEL_ID" \
```

Shellcheck (0.11.0)

[warning] bamboo_BEDROCK_MODEL_ID is referenced but not assigned.

(SC2154)